### ISSUE TITLE
Save ViewPoint

### ISSUE THREAD

**Issue by pnrobinson** [TheJacksonLaboratory/VPV#7]
We need to mark whether the user has worked on a ViewPoint (e.g., after they use the Close button) so that they will not necessarily open the same viewpoint again.

**Comment by pnrobinson** [comment]
On second thought, I do not think this is a good feature, since it would complicate things a lot and in principle users only need to go through and either add at least one fragment or to delete the viewpoint, so we can easily see viewpoints that need work just by looking for viewpoints with no fragments -- this is easy to do in the current version.

**Comment by hansenp** [comment]
But the feature might be good for documentation purposes. When I looked through existing Capture-(Hi-)C publications I saw that people indicate, whether a viewpoint was derived bioinformatically by certain rules, or edited manually. See here: https://images.nature.com/original/nature-assets/ng/journal/v46/n2/extref/ng.2871-S1.pdf
in Table S1 column 3.

Therefore, I introduced the variable _derivationApproach_.
